Projects/NEWS API/NEWS API.py

```diff
@@ -0,0 +1,120 @@
+    speak = Dispatch("SAPI.SpVoice")
+    speak.Speak(msg)
+
+
+# This will record our movements in program
+def login(msg):
+    global name
+    with open("record.txt", "a") as f:
+        f.write(f"{name.capitalize()} has {msg} on {datetime.now()}\n")
+
+
+if __name__ == '__main__':
+    while True:
+        print("\nWelcome to News API made by Sachin Dabhade\n")
+        name = input("Enter your name: ")
+        print("\nHello " + name.capitalize() + "! Best of Luck!")
+        print("\nThe News API is starting...!\n Let's Enjoy the programme...!\n\n")
+        login("started News API")
+
+        # Country names and category is fetching
+        countrys = {'Argentina': 'ar', 'Australia': 'au', 'Austria': 'at', 'Belgium': 'be', 'Brazil': 'br',
+                    'Bulgaria': 'bg', 'Canada': 'ca', 'China': 'cn', 'Colombia': 'co',
+                    'Cuba': 'cu', 'Czech Republic': 'cz', 'Egypt': 'eg', 'France': 'fr', 'Germany': 'de', 'Greece': 'gr',
+                    'Hong Kong': 'hk', 'Hungary': 'hu', 'India': 'in',
+                    'Indonesia': 'id', 'Ireland': 'ie', 'Israel': 'il', 'Italy': 'it', 'Japan': 'jp', 'Latvia': 'lv',
+                    'Lithuania': 'it', 'Malaysia': 'my', 'Mexico': 'mx',
+                    'Morocco': 'ma', 'Netherlands': 'nl', 'New zealand': 'nz', 'Nigeria': 'ng', 'Norway': 'no',
+                    'Philippines': 'ph', 'Poland': 'pl', 'Portugal': 'pt',
+                    'Romania': 'ro', 'Russia': 'ru', 'Saudi arabia': 'sa', 'Serbia': 'rs', 'Singapore': 'sg',
+                    'Slovakia': 'sk', 'Slovenia': 'si', 'South africa': 'za',
+                    'South korea': 'kr', 'Sweden': 'se', 'Switzerland': 'ch', 'Taiwan': 'tw', 'Thailand': 'th',
+                    'Turkey': 'tr', 'UAE': 'ae', 'Ukraine': 'ua',
+                    'United kingdom': 'gb', 'United states': 'us', 'Venuzuela': 've'}
+        loop(countrys)
+        country = input("\n\nEnter the Country (EX. India):")
+
+        # Checking the input category is valid or not
+        if country.capitalize() not in countrys.keys():
+            print("Invalid input....! Please try again....!")
+            speak("Invalid input....! Please try again....!")
+            exit()
+
+        country1 = countrys.get(country.capitalize())
+
+        # Enter the category that you like to here
+        categories = ['business', 'entertainment', 'health', 'science', 'sports', 'technology']
+        loop(categories)
+        category = input("\n\nEnter the category (EX. Science):")
+
+        # Checking the input category is valid or not
+        if category.lower() not in categories:
+            print("Invalid input....! Please try again....!")
+            speak("Invalid input....! Please try again....!")
+            exit()
+
+        # url from which the news will fetch
+        url = (
+            f'https://newsapi.org/v2/top-headlines?country={country1.lower()}&category={category.lower()}&apiKey=2114b271bda54165b23a311b1d1e3c49')

```

Comment:
API key exposed, please use `.env` to store credentials.
Refer to this blog to learn more about the environment variables 😇 
https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1